code/stable/dblpend/SRS/HTML/DblPend_SRS.html

```diff
@@ -422,16 +428,16 @@ <h2>Abbreviations and Acronyms</h2>
       <div class="section">
         <h1>Introduction</h1>
         <p class="paragraph">
-          A pendulum consists of mass attached to the end of a rod and its moving curve is highly sensitive to initial conditions. Therefore, it is useful to have a program to simulate the motion of the pendulum to exhibit its chaotic characteristics. The program documented here is called Pendulum.
+          A pendulum consists of mass attached to the end of a rod and its moving curve is highly sensitive to initial conditions. Therefore, it is useful to have a program to simulate the motion of the pendulum to exhibit its chaotic characteristics. The program documented here is called Double Pendulum.
         </p>
         <p class="paragraph">
-          The following section provides an overview of the Software Requirements Specification (SRS) for Pendulum. This section explains the purpose of this document, the scope of the requirements, the characteristics of the intended reader, and the organization of the document.
+          The following section provides an overview of the Software Requirements Specification (SRS) for Double Pendulum. This section explains the purpose of this document, the scope of the requirements, the characteristics of the intended reader, and the organization of the document.
         </p>
```

Comment:
Up to this point we call the program Double Pendulum.  From this point forward we call it DblPend.  Unfortunately, it doesn't look like we introduce this short form (abbreviation).  We are treating Double Pendulum as the program name in the line above, but we call it DblPend below.  What is the program name?  I feel like the simple fix would be to make the program name DblPend, and never call it Double Pendulum.

Reply:
There is already an issue regarding this, see #3529

Reply:
Okay.  We can handle this with issue #3529.

Reply:
let me know if I should change the program name to be DblPend anyways

Reply:
No its fine as it is.  This PR is confusing enough.  :smile: